src/pubtools/sign/signers/msgsigner.py

```diff
@@ -132,12 +132,17 @@ class MsgSigner(Signer):
     ]
 
     def _construct_signing_message(
-        self: MsgSigner, claim, signing_key, extra_attrs: Optional[Dict] = None
+        self: MsgSigner,
+        claim,
+        signing_key,
+        extra_attrs: Optional[Dict] = None,
+        sig_type: str = "container_signature",
     ):
+        data_attr = "claim_file" if sig_type == "container_signature" else "data"
```

Comment:
This seems like an odd way to implement it, Maybe sig_type should be a boolean?

Reply:
Why do you find it strange? sig_type is directly used as mtype in messaging headers. Also bool wouldn't allow to implement potential next signature type in the future

Reply:
If multiple types will be allowed, shouldn't it be an enum? You can't just set it to any string and expect it to work, right? The possible values are not documented anywhere either.